library/cartridge_set_package_info.py

```diff
@@ -94,29 +94,61 @@ def get_deb_info(package_path):
 
     return {
         'name': package_name,
-        'tnt_version': tnt_version
+        'tnt_version': tnt_version,
+    }
+
+
+def get_tgz_info(package_path, app_name):
+    with tarfile.open(package_path) as tar:
+        try:
+            tar.getmember(app_name)
+        except KeyError:
+            raise Exception("Package should contain %s directory" % app_name)
+
+        version_file_path = os.path.join(app_name, 'VERSION')
+        try:
+            member = tar.getmember(version_file_path)
+        except KeyError:
+            raise Exception("Package should contain %s file" % version_file_path)
+
+        version_file = tar.extractfile(member)
+        version_file_lines = version_file.readlines()
+
+    tnt_version = None
+    for line in version_file_lines:
+        m = re.search(r'TARANTOOL=(\d+\.\d+)\.', line.decode())
+        if m is not None:
+            tnt_version = m.groups()[0]
+            break
```

Comment:
```suggestion
    tnt_version = None
    for line in version_file_lines:
        m = re.search(r'TARANTOOL=(\d+\.\d+)\.', line.decode())
        if m is not None:
            tnt_version = m.groups()[0]
            break
    
    if tnt_version is None:
        raise Exception("No Tarantool version in file '%s'", version_file_path)
```

Reply:
It's a valid case - in case of using Tarantool Enterprise VERSION file contains TARANTOOL_SDK key but doesn't contain TARANTOOL.